src/platform/tigerlake/include/platform/lib/memory.h

```diff
@@ -263,8 +263,8 @@
 	HEAP_RT_COUNT1024 * 1024 + HEAP_RT_COUNT2048 * 2048 + \
 	HEAP_RT_COUNT4096 * 4096)
 
-#define HEAP_BUFFER_SIZE	0xF000
-#define HEAP_BUFFER_BLOCK_SIZE		0x180
+#define HEAP_BUFFER_SIZE	0x50000
+#define HEAP_BUFFER_BLOCK_SIZE		0x100
```

Comment:
We should make the block size consistent across all platforms

Reply:
Also is there enough memory for that significant increase (from 60k to 320k)?

Reply:
@paulstelian97 AFAIK there is still memory left as TGL has ~3MB local SRAM. @mrajwa Do you know what is approx. safety margin you have with this HEAP_BUFFER_SIZE for requested typologies ?

Reply:
@lbetlej, yes this PR changes the HEAP_BUFFER_SIZE to reasonable size which is about 10% of whole memory and allows us to load multi pipeline topology including WoV. We still have plenty of room available so there is no reason to worry about boundaries at this point of time as we are not even close to them.

Reply:
That's what I wanted to hear, that there is a 3MB total.

On i.MX8* I have an 8MB range for the heaps. It's got a few blocks (I'll add a few 4kB blocks soon on one of the platforms) and everything else in that 8MB range will be buffer heaps. Essentially the memory map already has reserved a large enough chunk that there isn't need for more in the foreseeable future. When I need something in a heap other than the buffer heap, I just move blocks around.